build.csx

```diff
@@ -0,0 +1,225 @@
+#load "packages/simple-targets-csx.5.2.0/simple-targets.csx"
+
+#r "System.Xml.Linq"
+
+using System.Runtime.Serialization.Json;
+using System.Text.RegularExpressions;
+using System.Xml;
+using System.Xml.Linq;
+using static SimpleTargets;
+
+static var solutionName = "FakeItEasy";
+
+static var solution = "./" + solutionName + ".sln";
+static var packagesDir = Path.GetFullPath("packages");
+var repoUrl = "https://github.com/FakeItEasy/FakeItEasy";
+var assemblyInfo = "./src/CommonAssemblyInfo.cs";

```

Comment:
Why bother adding `static` at all? It just adds noise, and it a micro-optimisation at best.